### PR TITLE
Update docker deployment images

### PR DESCRIPTION
**Pull request by MNie**
Update docker deployment images so they align to the current `design` of Azure :)

CC: @theimowski

### CODE REVIEW

File: docs/template-docker.md [hunk 1]
```diff
@@ -51,13 +51,15 @@ This will build the server and client, run all test, put the app into a docker c
 ### Azure Portal
 
 Go to the [Azure Portal](https://portal.azure.com) and create a new "Web App for Containers".
-Configure the Web App to point to the docker repo and select `latest` channel of the container.
+Configure the Web App to point to the docker repository and select image with tag.
```
Comment (theimowski) [inline docs/template-docker.md]: "docker public repository and type in image and tag."

File: docs/template-docker.md [hunk 2]
```diff
@@ -51,13 +51,15 @@ This will build the server and client, run all test, put the app into a docker c
...
-![Docker setup](https://user-images.githubusercontent.com/57396/31279587-e06001d0-aaa9-11e7-9b4b-a3e8278a6419.png)
+![](img/dockersetup.png)
 
-Also look for the "WebHook Url" on the portal, copy that url and set it as new trigger in your Docker Hub repo.
+"WebHook Url" to trigger new deployment after new image comes into your Docker Hub repo. Is available in `Settings/Container Settings` of your deployed app
 
-*Note that entering a Startup File is not necessary.*
+![](img/dockerwebhook.png)
+
+*Note that startup command is not necessary.*
```
Comment (theimowski) [inline docs/template-docker.md]: Startup Command or Startup File? Screenshot shows "Startup File"

File: docs/template-docker.md [hunk 3]
```diff
@@ -51,13 +51,15 @@ This will build the server and client, run all test, put the app into a docker c
...
 
 Go to the [Azure Portal](https://portal.azure.com) and create a new "Web App for Containers".
-Configure the Web App to point to the docker repo and select `latest` channel of the container.
+Configure the Web App to point to the docker repository and select image with tag.
 
-![Docker setup](https://user-images.githubusercontent.com/57396/31279587-e06001d0-aaa9-11e7-9b4b-a3e8278a6419.png)
+![](img/dockersetup.png)
 
-Also look for the "WebHook Url" on the portal, copy that url and set it as new trigger in your Docker Hub repo.
+"WebHook Url" to trigger new deployment after new image comes into your Docker Hub repo. Is available in `Settings/Container Settings` of your deployed app
```
Comment (theimowski) [inline docs/template-docker.md]: Let's leave the note to copy url to a trigger in Docker Hub. "Is available" -> "It is available"

**Comment by theimowski** [comment]
Just a couple minor comments to correct

File: docs/template-docker.md (hunk 2 above)
Comment (MNie) [reply docs/template-docker.md]: image
:P

**Comment by theimowski** [comment]
thanks!

**Pull request merged**